cmd/occollector/app/collector/collector.go

```diff
@@ -0,0 +1,259 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package collector handles the command-line, configuration, and runs the OC collector.
+package collector
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/census-instrumentation/opencensus-service/cmd/occollector/app/builder"
+	"github.com/census-instrumentation/opencensus-service/exporter"
+	"github.com/census-instrumentation/opencensus-service/exporter/exporterparser"
+	"github.com/census-instrumentation/opencensus-service/internal/collector/jaeger"
+	"github.com/census-instrumentation/opencensus-service/internal/collector/opencensus"
+	"github.com/census-instrumentation/opencensus-service/internal/collector/processor"
+	"github.com/census-instrumentation/opencensus-service/internal/collector/zipkin"
+	"github.com/spf13/cobra"
```

Comment:
what is the project convention for organizing imports? typical is to put the non census imports above and separate section for census ones

Reply:
I was letting the IDE automatically add them in alphabetical order but the convention is what you described.